xtas/tasks/single.py

```diff
@@ -30,7 +30,7 @@
 
 from cytoolz import identity, pipe
 import nltk
-import spotlight
```

Comment:
Local imports are lazy. They're used when importing causes work to be performed (e.g. starting a Stanford NER wrapper Java process thingy).